cypress/integration/tests/newShopping.spec.js

```diff
@@ -28,46 +28,34 @@ describe("Should test shopping flow", () => {
         Cypress.Cookies.preserveOnce(testData.cookies.cartItems)
     })
 
-    // it('should conduct search,open the first product, recommend to a friend, choose color, size, and add to cart', () => {
-    //     homePage.conductSearch()
-    //     searchResultPage.validateCorrectTermIsSearched()
-    //     searchResultPage.openFirstProduct()
-    //     productPage.validateCorrectProductIsOpened()
-    //     productPage.validateProductPricesMatch()
-    //     productPage.conductSendToAFriendAndValidate()
-    //     productPage.chooseColorSizeAddToCartAndValidate()
-    //     addToCartModal.continueShoppingAndValidate()
-        
-
-    // })
-
-    it('should check item is added and go to dresses', () => {
-        homePage.clickDressesLinkAndValidate(0)
+    it('should conduct search,open the first product, recommend to a friend, choose color, size, and add to cart', () => {
+        homePage.conductSearch()
+        searchResultPage.validateCorrectTermIsSearched()
+        searchResultPage.openFirstProduct()
+        productPage.validateCorrectProductIsOpened()
+        productPage.validateProductPricesMatch()
+        productPage.conductSendToAFriendAndValidate()
+        productPage.chooseColorSizeAddToCartAndValidate()
+        addToCartModal.continueShoppingAndValidate()
```

Comment:
Just observation, not something you have to change: 
You decided to make a separate page for the card modal. I put that on the product page.

Reply:
ce vidis zasto hahahaha